2-mandatory/1-create-functions.js

```diff
@@ -24,7 +25,10 @@ Write a function that:
 - Removes any forward slashes (/) in the strings.
 - Makes the strings all lowercase.
 */
-function tidyUpString() {
+function tidyUpString(arrayOfString) {
+return [...arrayOfString].map ((string) =>
+string.trim().replace("/", '').toLowerCase()
+);
 }
 
```

Comment:
you don't need to make another function inside a function, use the simple trim() replace() and regExp method's without the use of spread operator. Check my code which I have solved the problem in simpler way